### ISSUE TITLE
implement a physics subsystem

### ISSUE THREAD

**Issue by punytroll** [punytroll/galactic-fall#11]
This issue serves as a brain dump for all things related to the future physics subsystem.

Currently the positions and orientations are accessible through the VisualizationPrototype of the respective class. But Markers are specifically NOT a visual aspect.

Extract the physical properties of meshes from the littlem file into a separate description file which can be loaded separately into the physics subsystem.

For the moment this should include:
- slot positions and orientations
- mesh positions and orientations in the model
- exhaust positions, radii and orientations in the ship class
- muzzle positions

Basically the physics system is always involved, whenever something moves or turns.
But what about particle systems?
